hw/ip/sysrst_ctrl/doc/_index.md

```diff
@@ -0,0 +1,95 @@
+- Configuration registers can be set and locked until the next chip reset
+- Pin output override
+
+## Description
+
+The sysrst_ctrl logic is very simple. It looks up the configuration registers to decide how long the EC reset pulse duration and how long the keyboard debounce timer should be. Also what actions to take(eg. Interrupt, EC reset, GSC reset, disconnect the battery from the power tree).
+
+## Compatibility
+
+The configuration programming interface is not based on any existing interface.
+
+# Theory of Operations
+
+The block diagram shows a conceptual view of the sysrst_ctrl building blocks. There are 3 main modules. The first is the configuration and status registers. The second is the keyboard combo debounce and detection logic. The third is the pinout override logic.
+
+The sysrst_ctrl has four input pins (pwrb_in, key0,1,2_in) with corresponding output pins (pwrb_out, key0,1,2_out). During normal operation the sysrst_ctrl will pass the pin information directly from the input pins to the output pins with optional inversion. Combinations of the inputs being active for a specified time can be detected and used to trigger actions. The override logic allows the output to be overridden (i.e. not follow the corresponding input) based either on trigger or software settings. This allows the security chip to take over the inputs for its own use without disturbing the main user.
+
+The sysrst_ctrl also controls an active low open drain reset I/O ec_rst_l (typically used for the embedded controller reset). This output is always asserted when the IP block is reset (allowing its use as a power-on reset) and remains asserted until released by software. A pulse width is configured for the reset signal. Trigger actions may cause the sysrst_ctrl to assert ec_rst_l for the specified width. If the ec_rst_l is asserted by some other agent in the system then the sysrst_ctrl will extend the assertion to ensure the minimum pulse width is generated.
+
+## Block Diagram
+
+![sysrst_ctrl Block Diagram](sysrst_ctrl_overview.svg)
+
+## Combo detection
+
+Let’s use the Power button+Esc+Refresh combo as an example
+The SW defines three keys(eg. pwrb==0 && key0_in==0 && key1_in==0) as trigger in the configuration(com0_sel_ctl)
+The SW defines the duration(eg 10 seconds) that the above combo should be pressed in the configuration(com0_det_ctl)
+The SW defines the actions such as asserting ec_rst_l and raising interrupt in the configuration(com0_out_ctl)
+The SW defines the ec_rst_l pulse width in the configuration(ec_rst_ctl)
+The SW locks the configuration(no write is allowed) with cfg_wen_ctl
+sysrst_ctrl will detect the high(logic 1) to low(logic 0) transition on the combo and start the timer. If the timing condition is met(10 second), sysrst_ctrl will assert ec_rst_l,  sysrst_ctrl_intr and set the interrupt status register(intr_status) with the source
+The interrupt handler will read intr_status and write “logic 1” to clear the interrupt.
```

Comment:
`“”` are non ASCII characters (there are more further below)

can you replace those with `""`?

Reply:
Sure.